src/rasenmaeher_api/web/api/people/views.py

```diff
@@ -0,0 +1,66 @@
+"""People API views."""  # pylint: disable=too-many-lines
+from typing import List, Union
+import logging
+
+
+from fastapi import APIRouter, Request, Depends, HTTPException
+
+from .schema import (
+    CallSignPerson,
+    PeopleListOut,
+)
+from ..middleware.mtls import MTLSorJWT
+from ....db import Person
+
+LOGGER = logging.getLogger(__name__)
+
+router = APIRouter(dependencies=[Depends(MTLSorJWT(auto_error=True))])
+
+
+async def check_management_permissions(
+    raise_exeption: bool = True, callsign: str = "", required_role: str = "admin"
+) -> Union[bool, None]:
+    """
+    Simple function to check if requester has requested role.
+    """
+
+    user = await Person.by_callsign(callsign=callsign)
+    is_admin = await user.has_role(role=required_role)
+
+    # Raise exeption if
+    if raise_exeption and is_admin is False:
+        _reason = "Error. User doesn't have required permissions. See system logs."
+        LOGGER.error(
+            "Missing role from user : '{}'. Required permissions that are missing : '{}'".format(
+                callsign, required_role
+            )
+        )
+        LOGGER.error("{}".format(_reason))
+        raise HTTPException(status_code=403, detail=_reason)
+
+    return is_admin
+
+
+@router.get("/list", response_model=PeopleListOut)
```

Comment:
Use ValidUser middleware for checking admin privileges, see /api/v1/check-auth/validuser/admin for more info